### DISCUSSION TITLE
Asking for a demo application.

### DISCUSSION THREAD

**Discussion by KabirHere** [AmrDeveloper/CodeView#39] [General]
I am trying to build an code editor so I searched it and found your repository that pretty much impressed me. You described all the thing about how this repository can be used. But without any demo application it's really hard for the developer like me to build an apps using your repository. Will you please create a demo application explaining how we can build code editor and how we can improve it.

**Comment by AmrDeveloper** [comment]
Hello @azclub-ltd,

The repository has a Demo application with 4 different languages, auto complete, auto pairs, snippets, search, undo and redo ...etc, you can clone the repository and easily run the app module to check them all, and for any feature that you think it did not convert well we can cover it with more doc and examples

Thanks
Amr Hesham

**Reply by KabirHere** [reply]
I'll definitely try.
